src/main/java/uk/nhs/hee/tis/trainee/forms/api/AdminLtftResource.java

```diff
@@ -117,4 +120,19 @@ ResponseEntity<LtftFormDto> approveLtft(@PathVariable UUID id)
     Optional<LtftFormDto> form = service.updateStatusAsAdmin(id, APPROVED, null);
     return ResponseEntity.of(form);
   }
+
+  /**
+   * Unsubmit the form with the given ID, must be associated with the user's local office.
+   *
+   * @param id The ID of the form to unsubmit.
+   * @return The unsubmitted form.
+   * @throws MethodArgumentNotValidException When the state transition was not valid.
+   */
+  @PutMapping("/{id}/unsubmit")
+  ResponseEntity<LtftFormDto> unsubmitLtft(@PathVariable UUID id, @RequestBody
+      LftfStatusInfoDetailDto detail)
+      throws MethodArgumentNotValidException {
+    Optional<LtftFormDto> form = service.updateStatusAsAdmin(id, UNSUBMITTED, detail);
```

Comment:
All that hard work on the service in the last PR pays off: no changes needed there at all 👏